### PR TITLE
Minor Table Fixes

### PR DESCRIPTION
**Pull request by tahaNusoft**
Table Fixes
Switch component (Owais)

### CODE REVIEW

File: src/components/Toggle/Toggle.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./Toggle.module.scss";
+import { Col, Row, Switch, Typography } from "antd";
+import classNames from "classnames";
+
+interface ToggleProps {
+  checked?: boolean;
+  onChange?: (i: boolean) => void;
+  label?: string;
```
Comment (m-ammarkhan) [inline src/components/Toggle/Toggle.tsx:9]: `label` is not assigned a default value. So, it should be a required prop.

File: src/components/Toggle/Toggle.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./Toggle.module.scss";
+import { Col, Row, Switch, Typography } from "antd";
+import classNames from "classnames";
+
+interface ToggleProps {
+  checked?: boolean;
+  onChange?: (i: boolean) => void;
```
Comment (m-ammarkhan) [inline src/components/Toggle/Toggle.tsx]: `onChange` is not assigned a handler. So, it should be a required prop.

File: src/components/Toggle/Toggle.tsx (hunk 1 above)
Comment (Owais531) [reply src/components/Toggle/Toggle.tsx:9]: Label is being used for conditional rendering, therefore it is set as optional

File: src/components/Toggle/Toggle.tsx (hunk 2 above)
Comment (Owais531) [reply src/components/Toggle/Toggle.tsx]: Same as above

**Review by Owais531 — APPROVED** [review]

File: src/components/Toggle/Toggle.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "d-flex justify-content-center align-items-center"
+  );
+
+  const toggleLabel = classNames(styles.toggleLabel);
+  return (
+    <Row gutter={10}>
+      <Col>
+        <Switch className={toggleClass} checked={checked} onChange={onChange} />
+      </Col>
+      <Col>{label && <Text className={toggleLabel}>{label}</Text>}</Col>
```
Comment (m-ammarkhan) [inline src/components/Toggle/Toggle.tsx]:
```
{label && <Col><Text className={toggleLabel}>{label}</Text></Col>}
```

if label is optional, ignore the col block as well.

File: src/components/Toggle/Toggle.tsx (hunk 1 above)
Comment (m-ammarkhan) [reply src/components/Toggle/Toggle.tsx:9]: Best, Thank you.

File: src/components/Toggle/Toggle.tsx (hunk 2 above)
Comment (m-ammarkhan) [reply src/components/Toggle/Toggle.tsx]: handler is required.

on toggle we always want to do something.

File: src/components/Toggle/Toggle.tsx (hunk 3 above)
Comment (Owais531) [reply src/components/Toggle/Toggle.tsx]: Done

File: src/components/Toggle/Toggle.tsx (hunk 2 above)
Comment (Owais531) [reply src/components/Toggle/Toggle.tsx]: Done

**Review by m-ammarkhan — APPROVED** [review]

**Review by zohaib-q1996 — APPROVED** [review]

**Pull request merged**